package.json

```diff
@@ -32,7 +32,7 @@
     "should": "^11.1.2"
   },
   "dependencies": {
-    "esdoc": "^0.5.2",
```

Comment:
Wouldn't it be better to use `^1.0.1`?
It's the earliest version that should work with the Gulp plugin.

https://github.com/esdoc/esdoc/blob/master/CHANGELOG.md#101-2017-07-30

*But, I honestly only tested with 1.0.4 myself!*

Reply:
Same reason here. It's just what I tested so it's what I put in.